config/manager/default-config/deployment.yaml

```diff
@@ -46,10 +53,13 @@ spec:
               name: dynamic-plugins-npmrc
               readOnly: true
               subPath: .npmrc
+            - mountPath: /opt/app-root/src/dynamic-plugins.yaml
+              subPath: dynamic-plugins.yaml
+              name: dynamic-plugins-conf
           workingDir: /opt/app-root/src
 
       containers:
-        - name: backstage-backend
+        - name: backstage-backend    # placeholder for 'backstage-backend'
           image: quay.io/janus-idp/backstage-showcase:next
```

Comment:
I assume this value is changed / overwritten in the downstream implementation, since we can't deploy janus-idp inside RHDH. But it seems we don't have a transformation rule for converting from upsream to downstream. Could this instead be defined as an env var in the operator bundle's CSV? then we could just have a different set of images in single place in downstream, rather than needing to grep for `image: ` in *.yaml and *.yml files ...

Reply:
Why transformation and grep?
We can  have dedicated downstream, RHDP configuration (for images and for other reasons, like Backstage app configuration) and maintain it independently.